src/assets/locales/en/main.json

```
@@ -1456,6 +1456,9 @@
       "pleaseEnterValidURL": "Please enter a valid URL",
       "error": "Error",
       "success": "Success",
+      "pending": "Subscription is Pending",
+      "active": "Subscription is Active",
+      "process": "Subscription is in Process",
```

Are you certain about in process? Never saw that in the app up to now.
```suggestion
      "progress": "Subscription is in Progress",
```

---

Yes, progress is the right interpretation